src/three/plugins/ImplicitTilingPlugin.js

```diff
@@ -0,0 +1,56 @@
+import {SUBTREELoader} from "./SUBTREELoader.js";
+
+export class ImplicitTilingPlugin {
+
+	constructor() {
+		this.name = 'IMPLICIT_TILING_PLUGIN';
+	}
+
+	init(tiles) {
+
+		this.tiles = tiles;
+
+	}
+
+	preprocessNode(tile, uri) {
+		if (tile.implicitTiling) {	//only for root
```

Comment:
This function should set `tile.__hasUnrenderableContent` to `true` so the rest of the tile traversal logic can work as expected. This also means that the external tile set detection condition in `requestTileContents` will have to be modified to check for the `/json$/` extension rather than checking the above flag